src/graphics/graphics_device.js

```diff
@@ -829,6 +894,9 @@ pc.extend(pc, function () {
             if (texture.autoMipmap && pc.math.powerOfTwo(texture._width) && pc.math.powerOfTwo(texture._height) && texture._levels.length === 1 && !texture._compressed) {
                 gl.generateMipmap(texture._glTarget);
             }
+
+            var stats = pc.Application._currentApplication.stats.vram;
```

Comment:
Just a note on engine architecture. Currently, everything in src\graphics can run with zero dependency on anything else except maybe src\core. This breaks that.